minimal/docker-compose.yml

```diff
@@ -53,4 +53,13 @@ services:
     volumes_from:
       - {{ project_name }}_web
     ports:
-      - "0.0.0.0:80:80"
\ No newline at end of file
+      - "0.0.0.0:80:80"
+
+  {{ project_name }}_certbot:
+    restart: always
+    build: ./compose/certbot/production
```

Comment:
Fixed that. Now there are separate environments for staging and production each with their own `.env` file too.